flink-tests/src/test/java/org/apache/flink/test/checkpointing/StateCheckpointedITCase.java

```diff
@@ -47,7 +47,7 @@
  * A simple test that runs a streaming topology with checkpointing enabled.
  *
  * The test triggers a failure after a while and verifies that, after completion, the
- * state defined with either the {@link ValueState} or the {@link Checkpointed}
+ * state defined with either the {@link ValueState} or the {@link org.apache.flink.streaming.api.checkpoint.ListCheckpointed}
```

Comment:
do not use the fully qualified class name here. (it's not required since you import the class anyway ;) )